feature-foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/AuthorsCarousel.kt

```diff
@@ -58,17 +59,22 @@ import com.google.samples.apps.nowinandroid.core.designsystem.icon.NiaIcons
 import com.google.samples.apps.nowinandroid.core.designsystem.theme.NiaTheme
 import com.google.samples.apps.nowinandroid.core.model.data.Author
 import com.google.samples.apps.nowinandroid.core.model.data.FollowableAuthor
+import com.google.samples.apps.nowinandroid.core.ui.TrackScrollJank
 
 @Composable
 fun AuthorsCarousel(
     authors: List<FollowableAuthor>,
     onAuthorClick: (String, Boolean) -> Unit,
     modifier: Modifier = Modifier
 ) {
+    val lazyListState = rememberLazyListState()
+    TrackScrollJank(lazyListState, "ForYou:AuthorsCarousel")
```

Comment:
About the naming here, for macrobenchmark you're using something like `forYou:Feed` with a non-capital `f` as a `testTag`. Can you keep these kinds of strings consistent throughout the project?